src/Components/Card/Card.jsx

```diff
@@ -4,17 +4,20 @@ import { BsHeart } from "react-icons/bs";
 import { FiMessageCircle } from "react-icons/fi";
 import { FiSend } from "react-icons/fi";
 import { BsBookmark } from "react-icons/bs";
+import { BsBookmarkCheckFill } from "react-icons/bs";
 import { RiMoreFill } from "react-icons/ri";
 import { MdDeleteOutline } from "react-icons/md";
 import logo from "../../Assets/Images/logo.png";
 import { useAuth } from "../../features/Auth/authSlice";
```

Comment:
you can use absolute import in your app by adding a jsconfig.json file. Relative imports can grow lengthy. Just a suggestion.